webapi/src/main/scala/org/knora/webapi/routing/admin/PermissionsRouteADM.scala

```diff
@@ -63,16 +64,81 @@ class PermissionsRouteADM(routeData: KnoraRouteData) extends KnoraRoute(routeDat
      * Returns the route.
      */
     override def knoraApiPath: Route =
-        getAdministrativePermission ~
+        getAdministrativePermissionProjectGroup ~
+        getAdministrativePermissionsForProject ~
+        getDefaultObjectAccessPermissionsForProject ~
+        getPermissionsForProject ~
         createAdministrativePermission ~
         createDefaultObjectAccessPermission
 
-    private def getAdministrativePermission: Route = path(PermissionsBasePath / Segment / Segment) { (projectIri, groupIri) =>
+
+    private def getAdministrativePermissionProjectGroup: Route = path(PermissionsBasePath / "ap" / Segment / Segment) {
```

Comment:
maybe `getAdministrativePermissionForProjectAndGroup` would be clearer?

Reply:
The `And` in `getAdministrativePermissionForProjectAndGroup`  might cause confusion. When one sees this method name, one would expect that this route gets the administrative permission for a project and also for a group then combines them. That is not actually the purpose of this route. It returns the administrative permissions for a group defined in a certain project. I renamed it to `getAdministrativePermissionForProjectGroup`, I guess it follows the naming conventions of the clean code principle.